atlasreader/atlasreader.py

```diff
@@ -673,21 +670,34 @@ def create_output(filename, atlas='all', voxel_thresh=1.96, cluster_extent=20,
                                   cmap=plotting.cm.cold_hot,
                                   output_file=glass_fname)
 
-    # get template image for plotting cluster maps
-    bgimg = nb.load(
-        resource_filename(
-            'atlasreader',
-            'data/templates/MNI152_T1_1mm_brain.nii.gz'
+    # Check if thresholded image contains only zeros
+    if thresh_img.get_data().sum():
+
+        # get cluster + peak information from image
+        clust_frame, peaks_frame = get_statmap_info(
+            stat_img, atlas=atlas, voxel_thresh=voxel_thresh,
```

Comment:
To your question about providing `stat_img` vs `thresh_img` here: I thought it might be nice to have both `create_output()` and `get_statmap_info()` as stand-alone functions. That is, functions that a user can just call in the course of their work and get reasonable, good outputs. In that case it would be nice to just be able to provide a simple `stat_img` to either. 

You are right, though: we're duplicating work in terms of thresholding + clustering the image in both functions, but I think for allowing simple access to both it might be worthwhile. If you disagree I'm happy for you to remove the `process_img()` call from `get_statmap_info()` and feed `thresh_img` here instead of `stat_img`. What do you think?